connector/connect/src/main/protobuf/spark/connect/types.proto

```diff
@@ -26,31 +26,46 @@ option java_package = "org.apache.spark.connect.proto";
 // itself but only describes it.
 message DataType {
```

Comment:
This is a severe breaking change, we need to make sure that at some point in time our protos become stable. In particular, here the order of the members changes.

Reply:
maybe also need to add some UDTs(like VectorUDT) in the future, but now SQL types were completed.